src/options.cpp

```diff
@@ -1333,6 +1360,21 @@ void show_options(bool ingame)
     delwin(w_options_tooltip);
 }
 
+template <typename maptype>
+void tile_loading_report(maptype tiletypemap, std::string label, std::string prefix) {
```

Comment:
Well, once performance hit are those copies here, especially the copy of the map.

Better to take that map as a const reference:
```C++
void tile_loading_report( maptype const & tiletypemap, std::string const & label, std::string const & prefix )
```